File: C-Sharp/FerryLegacy/Bookings.cs
using System.Collections.Generic;

namespace FerryLegacy
{
    public class Bookings
    {
        private readonly List<Booking> _bookings = new List<Booking>();

        public void Add(Booking booking)
        {
            _bookings.Add(booking);
        }

        public IEnumerable<Booking> All()
        {
            return _bookings;
        }
    }
}

File: java/src/Bookings.java
import java.util.ArrayList;
import java.util.List;

public class Bookings {
	private final List<Booking> _bookings = new ArrayList<Booking>();

	public void Add(Booking booking) {
		_bookings.add(booking);
	}

	public List<Booking> All() {
		return _bookings;
	}
}
